README.md

```diff
@@ -38,10 +38,12 @@ for (let workplace of workplaces) {
 
 * account
   * [`login(email, password)`](docs/account.md#loginemail-password)
+  * [`backofficeLogin()`](docs/account.md##backofficelogin)
   * [`signup(params)`](docs/account.md#signupparams)
   * [`forgot(email)`](docs/account.md#forgotemail)
   * [`retrieve()`](docs/account.md#retrieve)
   * [`contexts(params)`](docs/account.md#contextsparams)
+  * [`menuForContext(context)`](docs/account.md#menuforcontextcontext)
```

Comment:
Again, not an elegant or simple resource method. Recommend:

```js
deskbookers.account.menu(params)
```

Parameters can specify menu return types.